script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -267,7 +267,7 @@ And in exchange for that pain, I faintly see the 'lines' of the walls and the gr
 // Assuming mineral is the Modern Major General variety of all inanimate
 // _things_ rather than the scientific definition.
 "As I suspected.
- You're a living being after all, you wouldn't be able to comprehend the 'death' of something mineral.
+ You're a living being after all, you wouldn't be able to comprehend the 'death' of minerals.
```

Comment:
See comment, I feel that 'something mineral' is a bit less literal than a pure chemical mineral. I might even go further here and change it to `inorganic`.